web_favicon/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+====================
+Custom shortcut icon
+====================
+
+This module was written to allow you to customize your Odoo instance's shortcut
+icon (aka favicon). This is useful for branding purposes, but also for
+integrators who have many different Odoo instances running and need to see at a
+glance which browser tab does what.
+
+Configuration
+=============
+
+Upload your favicon (16x16, 32x32 or 64x64 pixels, 16 colors) on the company
```

Comment:
A bigger resolution than 64x64 can be also used (and it's advisable for Android or iOS webapps for example). Also 16 colors is an older specification. Now the range color can be wider. Some reference: http://stackoverflow.com/questions/2268204/favicon-dimensions